vendorconsent/tcf2/rangesection_test.go

```diff
@@ -24,9 +25,12 @@ func TestRangeSectionConsent(t *testing.T) {
 	// 		assertBoolsEqual(t, ok, consent.PurposeAllowed(consentconstants.Purpose(i)))
 	//	}
 
-	vendorsWithConsent := buildMap(2, 23, 42, 127, 128, 587, 612, 626)
+	vendorsWithConsent := buildMap(23, 42, 126, 127, 128, 587, 613, 626)
 	for i := uint16(1); i <= consent.MaxVendorID(); i++ {
 		_, ok := vendorsWithConsent[uint(i)]
+		if ok != consent.VendorConsent(i) {
+			fmt.Printf("Vendor: %d failed\n", i)
+		}
```

Comment:
The test will fail on line 34, this just prints out which vendor failed the check. Otherwise we don't know where where in the loop the test failed.

This could be fixed better using `"github.com/stretchr/testify/assert"`, but that should probably be its own PR.